pages/account/index.vue

```diff
@@ -0,0 +1,95 @@
+<script>
+import { mapGetters } from 'vuex';
+import { MANAGEMENT } from '@/config/types';
+import Loading from '@/components/Loading';
+import ResourceTable from '@/components/ResourceTable';
+
+export default {
+
+  components: {
+    Loading,
+    ResourceTable,
+  },
+
+  async fetch() {
+    this.rows = await this.$store.dispatch('management/findAll', { type: MANAGEMENT.TOKEN, opt: { force: true } });
+  },
+
+  data() {
+    return { rows: null };
+  },
+
+  computed: {
+    ...mapGetters({ t: 'i18n/t' }),
+
+    headers() {
+      return this.$store.getters['type-map/headersFor'](this.schema);
+    },
+
+    schema() {
+      const inStore = this.$store.getters['currentProduct'].inStore;
+
+      return this.$store.getters[`${ inStore }/schemaFor`](MANAGEMENT.TOKEN);
+    },
+
+    apiKeys() {
+      // Filter out tokens that are not API Keys
+      const isApiKey = (key) => {
+        const labels = key.metadata?.labels;
+        const kind = labels ? labels['authn.management.cattle.io/kind'] : '';
+
+        // Note: The ember ui looked for the label 'ui-session' - with Steve, this seems to npw
+        // be the label 'authn.management.cattle.io/kind'' with the value 'session
+
+        // TODO: Don't understand the Ember logic:
+        // return  ( !expired || !labels || !labels['ui-session'] ) && !current;
+        // For the Steve API, current always seems to be false, even for the current UI Session token
+        // Show the Token if is not a session token and its it not current
+        return kind !== 'session' && !key.current;
```

Comment:
The `authn.../kind` label is set by the API when it's derived from a login request (vs creating a standalone API key), but I don't think it's relevant to look at here unless you want like a "user session' vs "api key" type/icon column.

The Ember UI sets `ui-session: true` in the request to login, which apparently Vue does not do; So that's why it's missing when you looked.  This should be added for consistency;  https://github.com/rancher/dashboard/blob/master/store/auth.js#L223

The `expired` and `current` properties in the object are basically useless because they're computed things in Norman that do not have anything to compute them in Steve.

You can easily derive whether a token is expired by comparing `expiresAt` with now in dayjs again.  There is little reason to show expired UI sessions (and I believe they're automatically deleted by a controller after a short while nowadays  anyway)

You cannot determine `current`, without knowing the current session ID  Which is (for good reason) a httponly cookie that JS can't read.  The intention here was to prevent select all + delete from kicking you out by killing your current session.  But this either has to be dropped, or be reimplemented as a filter in Steve to set current=true when the token is the token that authorized the request.

Reply:
I'll take a look